test/conformance/start.sh

```diff
@@ -3,4 +3,5 @@
 set -euxo pipefail
 cd "$( dirname "${BASH_SOURCE[0]}" )/../.."
 
-docker-compose -f quickstart.yml -f quickstart-postgres.yml -f test/conformance/docker-compose.yml up "${1:-}" -d
+# shellcheck disable=SC2086
+docker-compose -f quickstart.yml -f quickstart-postgres.yml -f test/conformance/docker-compose.yml up ${1:-} -d
```

Comment:
I used double-quote, but it had unnecessary empty characters, so I disabled SC2086.
```
docker-compose -f quickstart.yml -f quickstart-postgres.yml -f test/conformance/docker-compose.yml up '' -d
```